server/public/js/common/application-service.js

```diff
@@ -18,20 +18,17 @@ export default class ApplicationService {
             complete: onCompletion,
             dataType: 'json',
             async: false
-        })
+        });
     }
 
     showDialog(msg) {
-        window.alert(createPrefixedString(msg));
+        window.alert('[' + this.prefix + '] ' + msg);
     }
 
     log(msg) {
-        console.log(createPrefixedString(msg));
+        console.log('[' + this.prefix + '] ' + msg);
     }
 
-    createPrefixedString(str) {
```

Comment:
For what it's worth my fix was to call:

this.createPrefixedString(msg)

Which do you prefer? I kind of liked the function but to be honest it's such a small detail, it doesn't matter.